doc/neps/nep-0028-website-redesign.rst

```diff
@@ -0,0 +1,302 @@
+===================================
+NEP 28 — numpy.org website redesign
+===================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Joe LaChance <joe@boldmetrics.com>
+:Status: Draft
+:Type: Informational
+:Created: 2019-07-16
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+NumPy is the fundamental library for numerical and scientific computing with
+Python. It is used by millions and has a large team of maintainers and
+contributors. Despite that, its `numpy.org <http://numpy.org>`_ website has
+never received the attention it needed and deserved. We hope and intend to
+change that soon. This document describes ideas and requirements for how to
+design a replacement for the current website, to better serve the needs of
+our diverse community.
+
+At a high level, what we're aiming for is:
+
+- a modern, clean look
+- an easy to deploy static site
+- a structure that's easy to navigate
+- content that addresses all types of stakeholders
+- Possible multilingual translations / i18n
+
+This website serves a couple of roles:
+
+- it's the entry point to the project for new users
+- it should link to the documentation (which is hosted separately, now on 
```

Comment:
Would a search box also within scope? One that would search both in scipy and numpy doc (or in an even wider ecosystem) would be even better.

Reply:
Good idea. Will add this. It's nice to have search.

Reply:
Note that the docs already have it, which is the most important part.

Reply:
But the plan is to separate the docs a little bit more from the main landing page. (given the affiliated package system in astropy, we were brainstorming about the idea to be able to use the same search box for multiple packages (as the user might not know our logic by which we separate out functionality to different packages), but that's not that big issue for numpy, neither can I offer a solution for it as we're not really there yet with implementation)

Reply:
I think Search box, that scan multiple packages will come up with the need of filter in the near future (means if user wants to search in specific package only). So I guess it will make it a bit complex.

Reply:
indeed it's complex, and might not make much sense for numpy, but nevertheless something to think about for the longer run.

Reply:
I added a search box in an "other aspects" section.